server/src/http/cloud/get_team_metadata.rs

```diff
@@ -45,9 +45,10 @@ pub async fn get_team_metadata(
             // Check if user has privileges to access this team
             let team_privileges = match db.get_privileges_by_team_id(&request.team_id).await {
                 Ok(privileges) => {
-                    if !privileges
```

Comment:
This might require a small hack. Later, we use the variable team_privileges to get the members of the team. In the scenario we are trying to fix here, the admin won't see himself as a member of the team. This will change later as he will be shown on the list the moment the first app gets registered. Do we care about this here, or do we want to patch it?